src/items/class-features/exocortex_mods_(ex).json

```diff
@@ -37,7 +37,7 @@
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.cM54UpNXExgDSbq4]{Exocortex}</p>\n<p>Your exocortex allows you to apply any one of the following drone mods to yourself as if you were a drone with that mod installed: armor slot, cargo rack, climbing claws, enhanced senses, hydrojets, jump jets, resistance, smuggler’s compartment, speed, or weapon proficiency (gaining proficiency in advanced melee or heavy weapons).</p>\n<p>@UUID[Compendium.sfrpg.class-features.h4EjHKa3CRq6CYuR]{Armor Slot (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.t1A7Tw7dNDTB5Pcs]{Cargo Rack (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.W49lpCz2CD7VwRB2]{Climbing Claws (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.q1Q4CghodW0XqoDk]{Enhanced Senses (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.4gjHCwW9idbr7gam]{Hydrojets (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.XhDmuUyUtVjgLrTo]{Jump Jets (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.E5UGSczcIiQjjPKI]{Resistance (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.vQZIgWmcaXkYi5sd]{Smuggler's Compartment (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.3YVIlvpNrqOZsWq8]{Speed (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.28PhX1gOjYNvC3Fa]{Weapon Proficiency (Ex; Exocortex)}</p>\n<p>At 11th level, you can apply an additional mod, and your options expand to include echolocators, excavator, flight system, greater resistance, invisibility field, and reactive camouflage. At 14th and 17th levels, you can apply an additional mod. Each time you gain a level, you can switch any of your mods for different mods, but you must always choose at least one mod from the 7th-level list.</p>\n<p>@UUID[Compendium.sfrpg.class-features.pqo5zoVm0SgSgpH9]{Echolocators (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.clyTaWVj65CVkLH2]{Excavator (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.NkWVPJFn6IGlc4Jh]{Flight System (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.NvU3xMfhhEBxAvbd]{Greater Resistance (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.4k95KNutF1eAghPx]{Invisibility Field (Ex; Exocortex)}</p>\n<p>@UUID[Compendium.sfrpg.class-features.0bdUwHd54TkeC62Z]{Reactive Camoflage (Ex; Exocortex)}</p>"
```

Comment:
Replace the original text of the mod with the link, same as the invisibility links